chart/templates/scheduler/scheduler-deployment.yaml

```
@@ -243,10 +243,10 @@ spec:
         {{- else if .Values.dags.gitSync.enabled }}
         - name: dags
           emptyDir: {}
-        {{- end }}
         {{- if and  .Values.dags.gitSync.enabled  .Values.dags.gitSync.sshKeySecret }}
```

```suggestion
        {{- if .Values.dags.gitSync.sshKeySecret }}
```

We are already in a gitsync enabled block.